bindata/network/openshift-sdn/rules.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+  prometheus: k8s
+  role: alert-rules
+  name: networking-rules
+  namespace: openshift-sdn
+spec:
+  groups:
+  - name: general.rules
+    rules:
+    - alert: NodeWithoutOVSPod
+      annotations:
+        message: |
+          All nodes should be running an ovs pod. One or more are not
+      expr: |
+        :kube_pod_info_node_count: != sum(kube_pod_container_info{namespace="openshift-sdn", pod=~"ovs.*"})
+      for: 1h
+      labels:
+        severity: warning
+    - alert: NetworkPodsCrashLooping
+      annotations:
+        message: Pod {{"{{"}} $labels.namespace{{"}}"}}/{{"{{"}} $labels.pod{{"}}"}} ({{"{{"}} $labels.container
+          {{"}}"}}) is restarting {{"{{"}} printf "%.2f" $value {{"}}"}} times / 5 minutes.
+      expr: |
+        rate(kube_pod_container_status_restarts_total{namespace="openshift-sdn"}[15m]) * 60 * 5 > 0
```

Comment:
this is already covered by the general alerts in the cluster monitoring stack: https://github.com/openshift/cluster-monitoring-operator/blob/5092b55844dfec03d61f0c767f01da887e95f40c/assets/prometheus-k8s/rules.yaml#L527-L535